File: Python/Program/PSU.py
from Komponen import Komponen

class Psu(Komponen):
    def __init__(self, merk, nama, dayaWatt, sertifikasi):
        super().__init__(merk, nama)
        self.dayaWatt = dayaWatt
        self.sertifikasi = sertifikasi
        
    def set_dayaWatt(self, dayaWatt):
        self._dayaWatt = dayaWatt
        
    def get_dayaWatt(self):
        return self._dayaWatt
    
    def set_sertifikasi(self, sertifikasi):
        self._sertifikasi = sertifikasi
        
    def get_sertifikasi(self):
        return self._sertifikasi

File: Java/Program/PSU.java
public class PSU extends Komponen {
    private int dayaWatt;
    private String sertifikasi;

    public PSU(String merk, String nama, int dayaWatt, String sertifikasi) {
        super(merk, nama);
        this.dayaWatt = dayaWatt;
        this.sertifikasi = sertifikasi;
    }

    public void setDayaWatt(int dayaWatt) {
        this.dayaWatt = dayaWatt;
    }

    public int getDayaWatt() {
        return dayaWatt;
    }

    public void setSertifikasi(String sertifikasi) {
        this.sertifikasi = sertifikasi;
    }

    public String getSertifikasi() {
        return sertifikasi;
    }
}